test/rng_testsuite/statistics_tests/exponential_distribution_test.pass.cpp

```diff
@@ -0,0 +1,638 @@
+// -*- C++ -*-
+//===-- exponential_distribution_test.cpp ---------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+//
+// Abstract:
+//
+// Test of exponential_distribution - check statistical properties of the distribution
+
+#include "support/utils.h"
+#include <iostream>
+
+#if TEST_DPCPP_BACKEND_PRESENT && TEST_UNNAMED_LAMBDAS
+#    include <CL/sycl.hpp>
+#    include <random>
+#    include <limits>
+#    include <oneapi/dpl/random>
+#    include <math.h>
+
+// Engine parameters
+constexpr auto a = 40014u;
+constexpr auto c = 200u;
+constexpr auto m = 2147483563u;
+constexpr auto seed = 777;
+
+template <typename ScalarRealType>
+int
+statistics_check(int nsamples, ScalarRealType lambda, const std::vector<ScalarRealType>& dpstd_samples)
+{
+    // theoretical moments
+    double tM = 1 / lambda;
+    double tD = 1 / (lambda * lambda);
+    double tQ = 9 / (lambda * lambda * lambda * lambda);
+
+    // sample moments
+    double sum = 0.0;
+    double sum2 = 0.0;
+    for (int i = 0; i < nsamples; i++)
+    {
+        sum += dpstd_samples[i];
+        sum2 += dpstd_samples[i] * dpstd_samples[i];
+    }
+    double sM = sum / nsamples;
+    double sD = sum2 / nsamples - sM * sM;
+
+    // comparison of theoretical and sample moments
+    double tD2 = tD * tD;
+    double s = ((tQ - tD2) / nsamples) - (2 * (tQ - 2.0 * tD2) / (nsamples * nsamples)) +
+               ((tQ - 3.0 * tD2) / (nsamples * nsamples * nsamples));
+
+    double DeltaM = (tM - sM) / sqrt(tD / nsamples);
+    double DeltaD = (tD - sD) / sqrt(s);
+
+    if (fabs(DeltaM) > 3.0 || fabs(DeltaD) > 3.0)
+    {
+        std::cout << "Error: sample moments (mean= " << sM << ", variance= " << sD
+                  << ") disagree with theory (mean=" << tM << ", variance= " << tD << ")";
+        return 1;
+    }
+
+    return 0;
+}
+
+template <class RealType, class UIntType>
+int
+test(oneapi::dpl::internal::element_type_t<RealType> lambda, int nsamples)
+{
+
+    sycl::queue queue(sycl::default_selector{});
+
+    // memory allocation
+    std::vector<oneapi::dpl::internal::element_type_t<RealType>> dpstd_samples(nsamples);

```

Comment:
I wouldn't suggest to use `dpstd_` in the code. This abbreviation is not actual anymore and it can confuse readers

Reply:
done